frontend/src/components/AddThreadForm/AddThreadForm.tsx

```diff
@@ -64,19 +64,26 @@ export const AddThreadForm: React.FC<AddThreadFormProps> = ({ show, setShowThrea
                                 <Row>
                                     <Col className="form-group mt-3" xl={12}>
                                         <label>Tytuł tematu:</label><br />
-                                        <Field className="form-control" type="text" name="threadTitle" />
-                                        <ErrorMessage name="threadTitle" component="div">
+                                        <Field className="form-control" type="text" name="name" />
+                                        <ErrorMessage name="name" component="div">
                                             {msg => <div style={{ color: 'red' }}>{msg}</div>}
                                         </ErrorMessage>
                                     </Col>
 
                                     <Col className="form-group mt-3" xl={12}>
                                         <label>Twoja wiadomość:</label>
-                                        <Field className="form-control" as="textarea" name="firstMsg" />
-                                        <ErrorMessage name="firstMsg" component="div">
+                                        <Field className="form-control" as="textarea" name="content" />
+                                        <ErrorMessage name="content" component="div">
                                             {msg => <div style={{ color: 'red' }}>{msg}</div>}
                                         </ErrorMessage>
                                     </Col>
+
+                                    <Col className="form-group mt-3" xl={12}>
+                                        <label>
+                                            Czy chcesz od razu zaobserwować ten temat?
+                                            <Field type="checkbox" name="isFavourite" className="mx-3" />
+                                        </label>
+                                    </Col>
```

Comment:
Tak swoją drogą, czy jak ktoś nie tworzy tematu, to nie powinien go z automatu obserwować?

Reply:
Ja bym to na razie zostawił, to nie jest jakiś duży problem moim zdaniem